### ISSUE TITLE
Simplify isochrones download/handling

### ISSUE THREAD

**Issue by Gabriel-p** [asteca/ASteCA#497]
- [x] Merge [CMDphotsysts](https://github.com/asteca/CMDphotsysts) into [ezpadova-2](https://github.com/asteca/ezpadova-2). The new `ezpadova-2` should store a file inside the isochrones folder with the information about that system's filters. For example:
```
PAR12+CS_37    Gmag  G_BPmag  G_RPmag   6437.70  5309.57  7709.85 3700  2080  2540
```
The folder's name should be simply the system's ID, all lower-case, i.e.: `gaiaedr3`
- [x] Re-arrange the way the photometric systems are recognized. From this:
```
#           mag               e_mag
I2        Gmag,28,Gmag       e_Gmag
#               col                e_col
I3   BP-RP,28,G_BPmag,G_RPmag    e_BP-RP
```
to this:
```
#           system    filter     mag_col      e_mag_col
I2        gaiaedr3      Gmag        Gmag         e_Gmag
#           system    filter1    filter2   color_col  e_color_col
I3        gaiaedr3    G_BPmag    G_RPmag       BP-RP      e_BP-RP
```
adding as many `I3` rows as colors desired. It is important that the system is converted to all lower-case to match the folder's name.
- [x] Remove the `evol_track` and similar parameters as they will be no longer needed.
- [x] Archive `CMDphotsysts` once everything else is done
